infra/utils.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing perepo_managerissions and
+# limitations under the License.
+"""Utilitys for OSS-Fuzz infrastrcture."""
+
+import os
+
+# Refrence to OSS-Fuzz home repo
+OSS_FUZZ_HOME = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Can you change helper.py, bisector.py, and any others in this dir to use this as well?

Reply:
@jonathanmetzman requested this in a separate  PR to make the git history neat.

Reply:
I think we probably want `abspath` instead of `realpath` here.